### DISCUSSION TITLE
Does mirrord support multiple developer working on same service at the same time ?

### DISCUSSION THREAD

**Discussion by AlwaysNoobCoder** [metalbear-co/mirrord#2222] [Q&A] [answered]
just wondering before try this, thanks.

**Comment by aviramha** [comment, marked as answer]
Hi @AlwaysNoobCoder, this is supported as part of [mirrord for Teams.](https://mirrord.dev/docs/overview/teams/)

**Reply by AlwaysNoobCoder** [reply]
Is  [mirrord for Teams.](https://mirrord.dev/docs/overview/teams/) free ?
@aviramha

**Reply by AlwaysNoobCoder** [reply]
never mind, I see pricing
